.github/workflows/test_ucxx.yaml

```diff
@@ -4,24 +4,26 @@ on:
   workflow_dispatch:
     inputs:
       branch:
+        required: true
         type: string
       date:
+        required: true
         type: string
       sha:
+        required: true
         type: string
       build_type:
         type: string
+        default: nightly
 
 jobs:
-  ucxx-python-tests:
+  conda-python-ucxx-tests:
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/conda-python-tests.yaml@branch-25.06
     with:
       build_type: ${{ inputs.build_type }}
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
-      script: ci/test_python.sh
-      env:
-        GH_TOKEN: ${{ github.token }}
-        GHA_DASK_UCXX_ONLY: true
+      gh_token: ${{ github.token }}
```

Comment:
```suggestion
```

This isn't a valid input on the `conda-python-tests.yaml` workflow here: https://github.com/rapidsai/shared-workflows/blob/1fdd8cfecadd8223a0741c19a1e76804a10b0f07/.github/workflows/conda-python-tests.yaml#L1-L33

Therefore we'll need to remove it.

Reply:
Good catch, removed.